scenarios/reproducers/shift-on-stack-overrides.yml

```diff
@@ -0,0 +1,6 @@
+cifmw_libvirt_manager_compute_disksize: 200
+cifmw_libvirt_manager_compute_memory: 50
+cifmw_libvirt_manager_compute_cpus: 8
+cifmw_allow_vms_to_reach_osp_api: true
+# HCI requires bigger size to hold OCP on OSP disks
+cifmw_block_device_size: 100G
```

Comment:
I think this isn't the right parameter name? Shouldn't it be something like:
```YAML
cifmw_libvirt_manager_compute_disksize: 100
```

Reply:
Yes, that variable is present in line 1. This is the size that the root disk will have: 200G.

Then, I need to set the variable **cifmw_block_device_size** which is the size for the virtual block device that will used by ceph on the computes: https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/cifmw_block_device/README.md#parameters

So, the outcome will be computes with 200G where 100G will be used by Ceph and the rest will be used by the compute itself.